public/content/en/welcome.md

```diff
@@ -90,13 +90,36 @@ On UNIX systems the shebang line `#!/usr/bin/env rdmd` can be put
 on the first line of an executable D file to allow a script-like
 usage.
 
-# Documentation & Links
+# Links & Documentation
 
-* API Documentation of standard library [Phobos](https://dlang.org/phobos)
-* [Language reference](https://dlang.org/spec/)
-* Repository of DUB libraries and applications: [code.dlang.org](http://code.dlang.org)
-* Free E-Book by Ali Çehreli: [Programming in D](http://ddili.org/ders/d.en/). Also
-  available as hardcover.
+The best places to learn D are
+
+* [Learning D as a first language](https://wiki.dlang.org/First_Language) - [As a second language](As a second language)
+* The [D Wiki](https://wiki.dlang.org/), which tends to contain the best information and links.
+* [Tutorials to D](https://wiki.dlang.org/Tutorials)
+* [Overview about D's unique features](http://dlang.org/overview.html)
+* [FAQ](http://dlang.org/faq.html)
+* [Articles about D](http://dlang.org/articles.html)
+* [Books about D](https://wiki.dlang.org/Books), for an more in-depth view
+
+## Help
```

Comment:
All sections must be third level. Second level is just allowed for source code. That's also the reason the Travis build failed